public/app/core/components/AppChrome/AppChromeService.tsx

```diff
@@ -56,6 +57,31 @@ export class AppChromeService {
     returnToPrevious: this.returnToPreviousData,
   });
 
+  public headerHeightObservable = this.state
```

Comment:
this `headerHeightObservable` is like a... sub-observable? It's not a new piece of state, but it's connected to the one 'global' state and only emits changes when it's portion changes?

Reply:
yep exactly. it will only emit changes if the actual value changes. my rxjs knowledge isn't amazing, so if there's a better pattern i can change this 🙇